Laba3\Laba4\Laba5

vanyc777 opened the pull request.

vanyc777 committed:
Laba3

vanyc777 committed:
Laba3

tapoton commented:
Графы:
В стране N городов, некоторые из которых соединены между собой дорогами. Для того, чтобы проехать по одной дороге, требуется один бак бензина. В каждом городе бак бензина имеет разную стоимость. Вам требуется добраться из первого города в N-ый, потратив как можно меньшее количество денег.
**Входные данные**
В первой сроке вводится число N (1<=N<=100), в следующей идет N чисел, i-ое из которых задает стоимость бензина в i-ом городе (все числа целые из диапазона от 0 до 100). Затем идет число M – количество дорог в стране, далее идет описание самих дорог. Каждая дорога задается двумя числами – номерами городов, которые она соединяет. Все дороги двухсторонние (то есть по ним можно ездить как в одну, так и в другую сторону); между двумя городами всегда существует не более одной дороги; не существует дорог, ведущих из города в себя.
**Выходные данные**
Требуется вывести одно число – суммарную стоимость маршрута или -1, если добраться невозможно.

vanyc777 committed:
Laba4

Графы:
В стране N городов, некоторые из которых соединены между собой дорогами. Для того, чтобы проехать по одной дороге, требуется один бак бензина. В каждом городе бак бензина имеет разную стоимость. Вам требуется добраться из первого города в N-ый, потратив как можно меньшее количество денег.
Входные данные
В первой сроке вводится число N (1<=N<=100), в следующей идет N чисел, i-ое из которых задает стоимость бензина в i-ом городе (все числа целые из диапазона от 0 до 100). Затем идет число M – количество дорог в стране, далее идет описание самих дорог. Каждая дор

vanyc777 committed:
Delete main.py

vanyc777 committed:
Delete input.txt

vanyc777 committed:
Laba4

Графы:
В стране N городов, некоторые из которых соединены между собой дорогами. Для того, чтобы проехать по одной дороге, требуется один бак бензина. В каждом городе бак бензина имеет разную стоимость. Вам требуется добраться из первого города в N-ый, потратив как можно меньшее количество денег.
Входные данные
В первой сроке вводится число N (1<=N<=100), в следующей идет N чисел, i-ое из которых задает стоимость бензина в i-ом городе (все числа целые из диапазона от 0 до 100). Затем идет число M – количество дорог в стране, далее идет описание самих дорог. Каждая дор

tapoton commented:
**Внешние сортировки:** В текстовом файле задана прямоугольная матрица. Отсортировать файл в порядке возрастания сумм первых и последних элементов строк матрицы.

vanyc777 committed:
Laba5

Внешние сортировки: В текстовом файле задана прямоугольная матрица. Отсортировать файл в порядке возрастания сумм первых и последних элементов строк матрицы.

vanyc777 committed:
Laba5

Внешние сортировки: В текстовом файле задана прямоугольная матрица. Отсортировать файл в порядке возрастания сумм первых и последних элементов строк матрицы.

vanyc777 committed:
Delete input.txt

vanyc777 committed:
Delete main.cpp
